app/init.php

```diff
@@ -529,12 +533,12 @@ function (mixed $value) {
             'multiple' => true,
             'schemes' => ['mariadb', 'mysql'],
         ],
-        'queue' => [
-            'type' => 'queue',
-            'dsns' => App::getEnv('_APP_CONNECTIONS_QUEUE', $fallbackForRedis),
-            'multiple' => false,
-            'schemes' => ['redis'],
-        ],
+//        'queue' => [
```

Comment:
We can remove this part and create an independent resource for the `queue`. Would be nice if we can have a similar preparation for using other adapters than Redis for future flexibility and extendability.